### PR TITLE
Referrals, cont.

### PR DESCRIPTION
**Pull request by bersace**
Hi @datinho, @cbandy,

Here is the following developments on @datinho work. Can you check this ?

cf. #236, #228

### CODE REVIEW

**Comment by bersace** [comment]
You can install the branch with

``` console
$ pip install https://github.com/bersace/ldap2pg/archive/referrals.zip
...
Successfully installed ldap2pg-4.12 psycopg2-2.7.5 pyasn1-0.4.4 pyasn1-modules-0.2.2 python-ldap-3.1.0 pyyaml-3.13
$ ldap2pg --version
ldap2pg 4.12
psycopg2 2.7.5 (dt dec pq3 ext lo64)
python-ldap 3.1.0
Python 3.5.3 (default, Jan 19 2017, 14:11:04)
[GCC 6.3.0 20170118]
$
```

File: ldaprc [hunk 1]
```diff
@@ -2,4 +2,4 @@ BINDDN  cn=admin,dc=ldap,dc=ldap2pg,dc=docker
 BASE    dc=ldap,dc=ldap2pg,dc=docker
 TLS_REQCERT allow
 NETWORK_TIMEOUT 5
-REFERRALS on
+REFERRALS off
```
Comment (cbandy) [inline ldaprc]: Do we need this line since the default is off/false?

File: ldap2pg/ldap.py [hunk 2]
```diff
@@ -210,10 +210,7 @@ def connect(**kw):
         conn = UnicodeModeLDAPObject(conn)
 
     conn = LDAPLogger(conn)
-
-    if not options.get('REFERRALS'):
-        logger.debug("HOTFIX https://stackoverflow.com/questions/18793040/python-ldap-not-able-to-bind-successfully")
-        conn.set_option(ldap.OPT_REFERRALS, 0)
+    conn.set_option(ldap.OPT_REFERRALS, options.get('REFERRALS', False))
```
Comment (cbandy) [inline ldap2pg/ldap.py]: This seems like a good place to leave a comment about why we touch this setting at all. (If you like that sort of thing in comments.)

The unclear error message would be good to include.

File: ldaprc (hunk 1 above)
Comment (bersace) [reply ldaprc]: The versionned ldaprc is here just for testing. This is alsow a showcase of ldaprc. Given @datinho added it, let's keep it for a while.

File: ldap2pg/ldap.py (hunk 2 above)
Comment (bersace) [reply ldap2pg/ldap.py]: You're right !

**Comment by datinho** [comment]
Tested the branch successfully.
I've tried in both case, with referrals on and off, the behavior is the expected.

**Comment by bersace** [comment]
@datinho awesome. Thanks !

**Pull request merged**